decentraland/screens/MyAccountScreen.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import MyAccount from "@presentational/MyAccount";
+import { connect } from "react-redux";
+import AccountCreationActions from "@constants/AccountCreationActions";
+import ActionStatus from "../constants/ActionStatus";
+import PropTypes from "prop-types";
+
+export class MyAccountScreen extends React.Component {
+  static navigationOptions = {
+    title: "My Account",
+  };
+
+  render() {
+    const { accountInfo } = this.props;
+    const creationActions = [];
+    creationActions.push({
+      name: "Account created",
+      action: null,
+      status: !accountInfo.account ? ActionStatus.MISSING : ActionStatus.DONE,
+    });
+    Object.keys(AccountCreationActions).forEach(action => {
+      const creationAction = {
+        name: AccountCreationActions[action].name,
+        action,
+      };
+      if (accountInfo.creationActions.hasOwnProperty(action)) {
+        creationAction.status = ActionStatus.DONE;
+      } else {
+        creationAction.status = ActionStatus.MISSING;
+      }
+      creationActions.push(creationAction);
+    });
+    return (
+      <MyAccount
+        progress={this._getPercentage(!!accountInfo.account, creationActions)}
+        creationActions={creationActions}
+      />
+    );
+  }
+
+  _getPercentage(isAccountCreated, accountActions) {
+    let percentage = isAccountCreated ? 0.2 : 0;
+    accountActions.forEach(accountAction => {
+      if (
+        accountAction.status === ActionStatus.DONE &&
+        AccountCreationActions.hasOwnProperty(accountAction.action)
+      ) {
+        percentage += AccountCreationActions[accountAction.action].percentage;
+      }
+    });
+    return Math.round(percentage);
```

Comment:
Maybe we should round it to the nearest 5 to be defensive, even though we know we should pick nice round numbers for this outside of this screen

Reply:
Well that depends, because you manually set percentage, and when you have to divide 80% on 3 equal parts, you won't get 100% so I've rounded it up.

Reply:
What I mean is that independent of the numbers we have chosen here, a rule that progress comes in increments of 5 seems like it could be a good rule.

Reply:
Rounding up works, and it was a good call ensuring it sums to 100%. Maybe we should say account having been created is 25% instead of 20% to make our lives easier.